docs/docusaurus/docs/cloud/setup/setup_cloud.md

```diff
@@ -0,0 +1,183 @@
+
+3. Run the following command in an empty base directory inside a Python virtual environment:
+
+    ```bash title="Terminal input"
+    pip install great_expectations
+    ```
+
+    It can take several minutes for the installation to complete.
+
+    If you've previously installed GX, run the following command to upgrade to the latest version:
+
+    ```bash title="Terminal input"
+    pip install great_expectations --upgrade
+    ```
+4. In Jupyter Notebook, run the following Python code to import the modules you'll use to test functionality:
+
+    ```python title="Jupyter Notebook"
+    import great_expectations as gx
+    import pandas as pd
+    import os
+    ```
+
+## Generate your user access token and copy your organization ID
+
+You'll need your user access token and organization ID to create your Data Context. Access tokens shouldn't be committed to version control software.
+
+1. In GX Cloud, click **Settings** > **Tokens**.
+
+2. In the **Access tokens** pane, click **Create user access token**.
+
+3. Complete the following fields:
+
+    - **Token name** - Enter a name for the token that will help you quickly identify it.
+
+    - **Role** - Select **Admin**. For more information about the available roles, click **Information** (?) .
+
+4. Click **Create**.
+
+5. Copy the access token and store it in a secure location. The token can't be retrieved after you close the dialog.
+
+6. Click **Close**.
+
+7. Copy the value in the **Organization ID** field and save it in the same location with your user access token.
+
+
+## Create a Data Context
+
+A Data Context provides the configurations and methods for GX Cloud components, including Data Sources, Expectations, Profilers, and Checkpoints.
+
+1. In Jupyter Notebook, copy the following code into a cell to instantiate the GX Cloud Data Context.
+
+    ```python title="Jupyter Notebook"
+    os.environ["GX_CLOUD_ACCESS_TOKEN"] = "<user_access_token>"
+    os.environ["GX_CLOUD_ORGANIZATION_ID"] = "<organization_id>"
+    context = gx.get_context()
+    ```
+2. Replace `user_access_token` and `organization_id` with the values you created and saved previously. See [Generate your user access token and copy your organization ID](#generate-your-user-access-token-and-copy-your-organization-id).
+
+3. Run the code.
+
+## Connect to a Data Source
+
+A Data Source provides a standard API for accessing and interacting with data from a data source system.
+
+In Jupyter Notebook, run the following code to connect to existing `.csv` NYC taxi trip data stored in the `great_expectations` GitHub repository:
+
+```python title="Jupyter Notebook"
+datasource_name = "Test"
+datasource = context.sources.add_pandas(datasource_name)
+asset_name = "Test"
+path_to_data = "https://raw.githubusercontent.com/great-expectations/gx_tutorials/main/data/yellow_tripdata_sample_2019-01.csv"
+asset = datasource.add_csv_asset(asset_name, filepath_or_buffer=path_to_data)
+batch_request = asset.build_batch_request()
+```
```

Comment:
My key review comment is that I think we should consider including screenshots of what the result of each workflow step looks like in the UI. This will help users connect the code they're running to what's actually happening in the Cloud.

I believe we raised the concern of a user feeling like they need to switch from Cloud to Jupyter, from Cloud to Jupyter too much. However, I think it's a much larger concern that the user could run through this guide and have absolutely zero understanding of what their code accomplished.

Reply:
@rachhouse often there is no result after the Python code is run, so screenshots would not be helpful. For example, nothing happens when you run the code to add the modules. Screenshots should only be used to convey information that is unclear. Screenshots lengthen a procedure unnecessarily and are a maintenance burden. Finally, this topic will be rewritten soon, so adding screen captures isn't worth the time or effort at this time.

Reply:
@kwcanuck After the Data Source and Data Assets are created, they should be visible in the Cloud UI. I believe the Expectations & Suite are also visible once created, but would need to double check.

What do you think about adding intermediary section(s) to direct the user to view the result of their code in the Cloud as they create the Data Source, Data Asset, and Expectation Suite?